docs/content/en/docs/getting-started/optional/cni.md

```diff
@@ -5,20 +5,19 @@ weight: 15
 aliases:
     /docs/reference/clusterspec/optional/cni/
 description: >
- EKS Anywhere cluster yaml cni plugin specification reference
+ EKS Anywhere cluster yaml CNI plugin specification reference
 ---
 
-### Specifying CNI Plugin in EKS Anywhere cluster spec
+### Specifying CNI Plugin in EKS Anywhere cluster yaml spec
 
 #### Provider support details
 |                | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
 |:--------------:|:-------:|:----------:|:-------:|:----------:|:----:|
 | **Supported?** |   ✓	    |     ✓      |   	 ✓   |     ✓      |  ✓   |
 
-EKS Anywhere currently supports two CNI plugins: Cilium and Kindnet. Only one of them can be selected
-for a cluster, and the plugin cannot be changed once the cluster is created.
-Up until the 0.7.x releases, the plugin had to be specified using the `cni` field on cluster spec.
-Starting with release 0.8, the plugin should be specified using the new `cniConfig` field as follows:
+EKS Anywhere officially supports Cilium for production level providers as a CNI plugin. The plugin cannot be changed once the cluster is created.
```

Comment:
Remove "production level" ... it's supported on all providers

We need to clarify "The plugin cannot be changed once the cluster is created". This is not true. A customer can replace the default EKS-A Cilium with another CNI such as Enterprise Cilium

Reply:
Alright, I clarified that they cannot replace the CNI by changing the cniConfig field and added a link to the custom CNI section on the same page.